libraries/rush-lib/src/logic/ProjectChangeAnalyzer.ts

```diff
@@ -315,8 +315,15 @@ export class ProjectChangeAnalyzer {
             )
           ]);
 
+          if (symlinks.size > 0) {
+            terminal.writeWarningLine(
```

Comment:
I'm curious, is there a reason why this is being added? It's good to support/at least write a warning, but I'm just wondering what the impetus for this is

Reply:
Someone encountered a crash from this case.